pkg/printers/internalversion/printers.go

```diff
@@ -112,6 +112,7 @@ func AddHandlers(h printers.PrintHandler) {
 		{Name: "Age", Type: "string", Description: metav1.ObjectMeta{}.SwaggerDoc()["creationTimestamp"]},
 		{Name: "IP", Type: "string", Priority: 1, Description: apiv1.PodStatus{}.SwaggerDoc()["podIP"]},
 		{Name: "Node", Type: "string", Priority: 1, Description: apiv1.PodSpec{}.SwaggerDoc()["nodeName"]},
+		{Name: "REFERENCE", Type: "string", Priority: 1, Description: "Pods reference by rs, rc, deploy."},
```

Comment:
Not only rs, rc, deploy, it can also be ds, etc. The description can be more brief and clear.